packages/mobile/src/invite/saga.ts

```diff
@@ -82,18 +83,36 @@ export function getInvitationVerificationFeeInWei() {
 }
 
 export async function generateLink(inviteCode: string, recipientName: string) {
-  const packageName = VersionCheck.getPackageName().replace(/\.debug$/g, '.integration')
-  const playStoreLink = await VersionCheck.getPlayStoreUrl({ packageName })
+  const AndroidPackageName = VersionCheck.getPackageName().replace(/\.debug$/g, '.integration')
+  const playStoreLink = await VersionCheck.getPlayStoreUrl({ AndroidPackageName })
   const referrerData = encodeURIComponent(`invite-code=${inviteCode}`)
   const referrerLink = `${playStoreLink}&referrer=${referrerData}`
-  const shortUrl = await dynamicLink(referrerLink)
-  const msg = i18n.t('sendFlow7:inviteSMS', {
+
+  let appStoreUrl
+  try {
+    appStoreUrl = await VersionCheck.getAppStoreUrl()
+  } catch (error) {
+    // can't get App Store url on Android
+    appStoreUrl = 'http://play.google.com/store/apps/details?id=org.celo.mobile.debug'.replace(
```

Comment:
Can you help me understand why we'd hit this error?

Reply:
The functions throws returns an error Android.